### ISSUE TITLE
Extension is not working anymore

### ISSUE THREAD

**Issue by tstirrat** [sandervspl/auctionoton#6]
```
Access to XMLHttpRequest at 'https://auctionoton-edge-api.vercel.app/api/item?id=42702&server_name=arugal&faction=alliance&amount=1' from origin 'https://www.wowhead.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

GET https://auctionoton-edge-api.vercel.app/api/item?id=42702&server_name=arugal&faction=alliance&amount=1 net::ERR_FAILED 404
```

**Comment by sandervspl** [comment]
@tstirrat Hi! That's strange. I can not reproduce the error and the requests are working for me. Are you still experiencing this issue?

It might've been a short outage at Vercel where this API is hosted

**Comment by tstirrat** [comment]
No, it's pretty persistent. It's been broken for weeks:

<img width="341" alt="Screenshot 2023-02-04 at 5 25 08 pm" src="https://user-images.githubusercontent.com/702990/216796928-ed536fab-a51c-4851-8c03-7cc3111b570c.png">
<img width="442" alt="Screenshot 2023-02-04 at 5 26 00 pm" src="https://user-images.githubusercontent.com/702990/216796929-16ceae1b-fc0a-4cb3-a64b-2043329a7b2c.png">

I've tried disabling ad blockers etc.

I'll try in incognito mode perhaps.

URL 404s for me opening directly: https://auctionoton-edge-api.vercel.app/api/item?id=42702&server_name=arugal&faction=alliance&amount=1

**Comment by tstirrat** [comment]
I just tried it in incognito, to rule out any other extentions interfering. Still broken with the 404

**Comment by sandervspl** [comment]
Very strange. I'll see what I can do! Thanks for reporting the issue

**Comment by sandervspl** [comment]
For some reason the extension on chrome extension store reverted to some older version of the code. No idea what happened there. I was probably running a developer version on my end, which is the reason I did not have encounter this issue until I reinstalled the extension from the chrome store today and encountered your same error.

I've pushed an update to Chrome's extension dashboard. This could take some time.
